packages/runtime/lib/runtime.js

```diff
@@ -129,7 +130,19 @@ class Runtime extends EventEmitter {
 
     this.#isProduction = this.#configManager.args?.production ?? false
     this.#servicesIds = config.services.map(service => service.id)
-    this.#workers.configure(config.services, this.#configManager.current.workers, this.#isProduction)
+
+    const workersConfig = []
+    for (const service of config.services) {
+      const count = service.workers ?? this.#configManager.current.workers
+      if (count > 1 && service.entrypoint && !features.node.reusePort) {
+        this.logger.warn(`entrypoint and reusePort on service "${service.id}" requires a single worker, setting workers to 1 instead of ${count}`)
```

Comment:
```suggestion
        this.logger.warn(`"${service.id}" is set as the entrypoint, but reusePort is not available in your OS; setting workers to 1 instead of ${count}`)
```